specification/synapse/data-plane/Microsoft.Synapse/preview/2021-07-01-preview/symsSync.json

```diff
@@ -0,0 +1,326 @@
+        "description": "Update files in Syms",
+        "x-ms-examples": {
+          "Register files in Syms": {
+            "$ref": "./examples/SynapseMetastore_Update.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "in": "body",
+            "name": "updateBody",
+            "required": true,
+            "description": "The update body",
+            "schema": {
+              "$ref": "#/definitions/updateObject"
+            }
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "Metadata Request Accepted Response.",
+            "schema": {
+              "$ref": "#/definitions/metadataRequestAcceptedResponse"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorContract"
+            }
+          }
+        }
+      }
+    },
+    "/synapsemetastore/delete": {
+      "put": {
+        "tags": [
+          "synapsemetastore"
+        ],
+        "operationId": "SynapseMetastore_Delete",
+        "description": "Remove files in Syms",
+        "x-ms-examples": {
+          "Register files in Syms": {
+            "$ref": "./examples/SynapseMetastore_Delete.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "in": "body",
+            "name": "deleteBody",
+            "required": true,
+            "description": "The delete body",
+            "schema": {
+              "$ref": "#/definitions/metadataDeleteObject"
+            }
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "Metadata Request Accepted Response.",
+            "schema": {
+              "$ref": "#/definitions/metadataRequestAcceptedResponse"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorContract"
+            }
+          }
+        }
+      }
+    },
+    "/synapsemetastore/getrequeststatus": {
+      "get": {
+        "tags": [
+          "synapsemetastore"
+        ],
+        "operationId": "SynapseMetastore_GetRequestStatus",
+        "description": "Gets status of a request",
+        "x-ms-examples": {
+          "Register files in Syms": {
+            "$ref": "./examples/SynapseMetastore_GetRequestStatus.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Request Success Response.",
+            "schema": {
+              "$ref": "#/definitions/successResponse"
```

Comment:
>successResponse [](http://example.com/codeflow?start=37&length=15)

I think the name can be improved
1. `SuccessResponse` sounds too general
2. It is kind of misleading. It is just stands for API response, and can contain "Failed" response too.

Reply:
Agreed. Changed it to metaStoreRequestSuccessResponse